receptorctl/receptorctl/cli.py

```diff
@@ -169,9 +169,15 @@ def reload(ctx):
     results = rc.simple_command(f"reload")
     if "Success" in results and results["Success"]:
         print(f"Reload successful")
+        sys.exit(0)
```

Comment:
technically this isn't needed, running off of the end of the program will automatically give it a return code of 0.

Reply:
good point, fixed